modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,213 @@
+----
++
+[source,terminal]
+----
+$ base64 <directory_path>/bond1.config
+----
+
+. Prepare the environment variables. Replace `<machine_role>` with the node role, such as `worker`, and replace `<interface_name>` with the name of your additional `br-ex` bridge name.
++
+[source,terminal]
+----
+$ export ROLE=<machine_role>
+----
++
+[source,terminal]
+----
+$ export INTERFACE=<interface_name>
+----
+
+. Create a machine configuration manifest file for configuring the network plugin by entering the following command in your terminal:
++
+[source,terminal]
+----
+$ oc create -f <machine_config_file_name>
+----
+
+. Define each interface definition in a machine configuration manifest file:
++
+.Example of a machine configuration file with definitions added for `bond1`, `eno1`, and `en02`
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 12-${ROLE}-sec-bridge-cni
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:;base64,<base-64-encoded-contents-for-bond1.conf>
+        path: /etc/NetworkManager/system-connections/bond1.nmconnection
+        filesystem: root
+        mode: 0600
+      - contents:
+          source: data:;base64,<base-64-encoded-contents-for-eno1.conf>
+        path: /etc/NetworkManager/system-connections/eno1.nmconnection
+        filesystem: root
+        mode: 0600
+      - contents:
+          source: data:;base64,<base-64-encoded-contents-for-eno2.conf>
+        path: /etc/NetworkManager/system-connections/eno2.nmconnection
+        filesystem: root
+        mode: 0600
+# ...
+----
+
+. Create a machine configuration manifest file that defines the existing static interface that hosts `br-ex1` on any nodes restarted on your cluster:
++
+.Example of a machine configuration file that defines `bond1` as the interface for hosting `br-ex1`
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 12-worker-extra-bridge
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - path: /etc/ovnk/extra_bridge
+          mode: 0420
+          overwrite: true
+          contents:
+            source: data:text/plain;charset=utf-8,bond1
+          filesystem: root
+----
+
+. Apply the machine-configuration to your selected nodes:
+
+[source,terminal]
+----
+oc create -f <machine_config_file_name>
+----
+
+. Create an Open vSwitch (OVS) bridge, `br-ex1`, on nodes by using the OVN-Kubernetes network plugin to create an `extra_bridge` file`. Ensure that you save the file in the `/etc/ovnk/extra_bridge` path of the host. The file must state the interface name that supports the additional bridge and not the default interface that supports `br-ex`, which holds the primary IP address of the node. 
```

Comment:
minor layout change - move this up to line 152 (above: `Create a machine configuration manifest file that defines the existing static interface that hosts `br-ex1` on any nodes restarted on your cluster:`).

Layout should be:
- create an OVS bridge (explaining process)
- example configuration definition showing we're selecting bond1
- define manifest with example (bond1) in path
- apply manifest
- verification

:+1:  Looks great besides order!
- apply the

Reply:
:heavy_check_mark: